pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -229,6 +234,21 @@ sub _centos_symlink_protection {
             }
         );
     }
+    if ($bluehost_ea4) {
+        $security_advisor_obj->add_advice(
+            {
+                'key'        => 'Apache_bluehost_provided_symlink_protection',
```

Comment:
It is a clone of the EA3 message, to the letter, with the one exception that the URL is different as the details in the go-link have diverged between EA3 and EA4.  With this in mind, do we still need a new key?  The key is used only once per EA release, and refers to the same message, and the same configuration (which is merely implemented differently), but differ in go-links only.